ONSdigital/eq-author
# Split surveys reducer into `meta` and `items`

### What is the context of this PR?

Quick bit of refactoring to split the `surveys` reducer into `meta` and `items`. Also did the same with the relevant actions.

Added some additional tests to increase coverage but not on the `items` bit as this is due a refactor anyway.

### How to review 

- Ensure all tests pass
- Ensure linting passes
- Ensure app works as expected
